Tests/JExtractSwiftTests/JNI/JNIClassTests.swift

```diff
@@ -177,19 +183,21 @@ struct JNIClassTests {
       detectChunkByInitialLines: 1,
       expectedChunks: [
         """
-        @_cdecl("Java_com_example_swift_MyClass_allocatingInit__")
-        func Java_com_example_swift_MyClass_allocatingInit__(environment: UnsafeMutablePointer<JNIEnv?>!, thisClass: jclass) -> jlong {
-          let self$ = UnsafeMutablePointer<MyClass>.allocate(capacity: 1)
-          self$.initialize(to: MyClass())
-          return Int64(Int(bitPattern: self$)).getJNIValue(in: environment)
+        @_cdecl("Java_com_example_swift_MyClass__00024init__JJ")
+        func Java_com_example_swift_MyClass__00024init__JJ(environment: UnsafeMutablePointer<JNIEnv?>!, thisClass: jclass, x: jlong, y: jlong) -> jlong {
+          let _result$ = UnsafeMutablePointer<MyClass>.allocate(capacity: 1)
+          _result$.initialize(to: MyClass.init(x: Int64(fromJNI: x, in: environment!), y: Int64(fromJNI: y, in: environment!)))
+          let _resultBits$ = Int64(Int(bitPattern: _result$))
+          return _resultBits$.getJNIValue(in: environment!)
         }
         """,
         """
-        @_cdecl("Java_com_example_swift_MyClass_allocatingInit__JJ")
-        func Java_com_example_swift_MyClass_allocatingInit__JJ(environment: UnsafeMutablePointer<JNIEnv?>!, thisClass: jclass, x: jlong, y: jlong) -> jlong {
-          let self$ = UnsafeMutablePointer<MyClass>.allocate(capacity: 1)
-          self$.initialize(to: MyClass(x: Int64(fromJNI: x, in: environment!), y: Int64(fromJNI: y, in: environment!)))
-          return Int64(Int(bitPattern: self$)).getJNIValue(in: environment)
+        @_cdecl("Java_com_example_swift_MyClass__00024init__")
+        func Java_com_example_swift_MyClass__00024init__(environment: UnsafeMutablePointer<JNIEnv?>!, thisClass: jclass) -> jlong {
+          let _result$ = UnsafeMutablePointer<MyClass>.allocate(capacity: 1)
+          _result$.initialize(to: MyClass.init())
+          let _resultBits$ = Int64(Int(bitPattern: _result$))
```

Comment:
hmm, how do we end up with both `_` and `$`, I guess the `$` would be our preferred "so we don't conflict with user identifiers". We may have had this issue before already though, so don't worry about it in this PR